src/edu/stanford/dlss/was/WasapiDownloaderSettings.java

```diff
@@ -5,46 +5,157 @@
 import java.io.IOException;
 import java.util.Properties;
 
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
 public class WasapiDownloaderSettings {
+  // a small inner class to represent exceptions in general when loading the config.  prevents callers from
+  // having to know too much detail about more granular exceptions that might get thrown when reading properties,
+  // parsing CLI args, etc.
+  class LoadSettingsException extends Exception {
```

Comment:
not sure how I feel about this inner exception class.  Reasoning as to why it isn't a separate (non-inner) class?